README.md

```diff
@@ -52,6 +52,7 @@ cargo install toml-cli
 cd ~/ && git clone --recursive https://github.com/Start9Labs/embassy-os.git
 cd embassy-os/backend/
 ./install-sdk.sh
+cd ~/.cargo/bin/ && sudo ./embassy-sdk init
```

Comment:
all this mess because embassy-sdk is relying on /etc directory 
```
Filesystem I/O Error: /etc/embassy: Permission denied (os error 13)
```
this file should be expected somewhere in users home directory space

Reply:
This has been fixed by [#1501](https://github.com/Start9Labs/embassy-os/pull/1501)